configure.ac

```diff
@@ -190,8 +190,8 @@ AC_ARG_ENABLE([address-sanitizer], AS_HELP_STRING([--enable-address-sanitizer],
               [enabled AddressSanitizer support for detecting a wide variety of \
                memory allocation and deallocation errors]), \
               [AC_DEFINE(HAVE_ADDRESS_SANITIZER, 1, [enable AddressSanitizer])
-              CFLAGS="$CFLAGS -fsanitize=address"
-              CXXFLAGS="$CXXFLAGS -fsanitize=address"
+              CFLAGS="$CFLAGS -fsanitize=address -static-libasan"
+              CXXFLAGS="$CXXFLAGS -fsanitize=address -static-libasan"
```

Comment:
why not 
> LIBS="-static-libasan $LIBS

Reply:
It doesn't belong in `LIBS`. That variable is for specifying libraries to link. libasan is automatically linked when passing `-fsanitize=address`. `-static-libasan` just says *how* to link it.